### PR TITLE
feat: Added CheckboxWrap and CheckboxGroupWrap

### PR DESCRIPTION
**Pull request by mcollovati**
Part of #1371

### CODE REVIEW

File: vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroupWrap.java [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+     *
+     * @param selection
+     *            item string representation
+     * @param others
+     *            other items string representation
+     */
+    public void selectItem(String selection, String... others) {
+        ensureComponentIsUsable();
+        selectItem(toCollection(selection, others));
+    }
```
Comment (caalador) [inline vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroupWrap.java:66]: I feel we should have different methods for single and multiselection so `selectItem(String selection)` and change this to
```suggestion
    public void selectItems(String... selection) {
        ensureComponentIsUsable();
        selectItem(Arrays.asList(selection));
    }
```

File: vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroupWrap.java [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+    /**
+     * Select items by client string representation.
+     *
+     * @param selection
+     *            items string representation
+     */
+    public void selectItem(Collection<String> selection) {
```
Comment (caalador) [inline vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroupWrap.java:74]:
```suggestion
    public void selectItems(Collection<String> selection) {
```
Items as it is targeting a collection

File: vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroupWrap.java [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+    /**
+     * Deselect items by client string representation.
+     *
+     * @param selection
+     *            item string representation
+     * @param others
+     *            other items string representation
+     */
+    public void deselectItem(String selection, String... others) {
```
Comment (caalador) [inline vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroupWrap.java:98]: Same as for select.

File: vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/checkbox/CheckboxWrapTest.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        Assertions.assertThrows(IllegalStateException.class, checkbox_::click);
+    }
+
+    @Test
+    void click_readOnly_throws() {
+        view.checkbox.setReadOnly(true);
+        Assertions.assertThrows(IllegalStateException.class, checkbox_::click);
+    }
+
+}
```
Comment (caalador) [inline vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/checkbox/CheckboxWrapTest.java:100]: new line

**Review by caalador — CHANGES_REQUESTED** [review]

File: vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroupWrap.java (hunk 2 above)
Comment (mcollovati) [reply vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroupWrap.java]: done

File: vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroupWrap.java (hunk 3 above)
Comment (mcollovati) [reply vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroupWrap.java]: done

File: vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/checkbox/CheckboxWrapTest.java (hunk 4 above)
Comment (mcollovati) [reply vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/checkbox/CheckboxWrapTest.java]: done

**Review by caalador — APPROVED** [review]

**Pull request merged**